code/modules/smithing/finished_items.dm

```diff
@@ -325,7 +324,7 @@
 	icon_prefix = "katana_smith"
 	overlay_state = "hilt_katana"
 	force = WEAPON_FORCE_SLASH_LARGE
-	wielded_mult = WEAPON_SLASH_TWOHAND_MULT
+	wielded_mult = 1.5
```

Comment:
Please make this a define so its easier to change later. You can change the original defined value if you want.